Content.Tests/DMProject/Tests/Builtins/text2ascii_char.dm

```diff
@@ -0,0 +1,3 @@
+﻿/proc/RunTest()
+	world << text2ascii_char("abcdéfg",6)
```

Comment:
This line doesn't do anything (there are no connected clients) and is unnecessary
```suggestion
```